examples/center.html

```diff
@@ -13,22 +13,16 @@
   <p>Use <code>Alt</code>+<code>Shift</code>+drag to rotate the map.</p>
```

Comment:
I think `<code>Alt+Shift+Drag</code>` is nicer than `<code>Alt</code>+<code>Shift</code>+drag` or some other variant.  I've updated things to be consistent in d3ff8d7fb0c59a8d12f94f208147b9dc2a3524fb.